src/main/kotlin/org/rust/lang/doc/RustDocPipeline.kt

```diff
@@ -67,3 +71,13 @@ private fun RustDocAndAttributeOwner.innerDocs(): Sequence<Pair<RustDocKind, Str
 
 private val RustMetaItemElement.docAttr: String?
     get() = if (identifier.text == "doc") litExpr?.stringLiteralValue else null
+
+private class RustDocMarkdownFlavourDescriptor(
+    private val gfm: MarkdownFlavourDescriptor = GFMFlavourDescriptor()
```

Comment:
For the reference, the precise flavor of markdown used in doc comments is unspecified atm: https://users.rust-lang.org/t/what-markdown-flavor-is-used-by-documentation-comments/6451